tests/RobotFramework/tests/mqtt/custom_sub_topics_tedge-mapper-az.robot

```diff
@@ -0,0 +1,29 @@
+*** Settings ***
+Documentation       Purpose of this test is to verify that tedge-mapper-az subscribes the topics that are configured as az.topics
+
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+
+Suite Setup         Custom Setup
+Suite Teardown      Get Logs
```

Comment:
To ensure the tests works well for other adapters (e.g. ssh, local (not just docker)), you can add a special Teardown keyword which undoes the tedge.toml setting.

```
Custom Teardown
    Get Logs
    Execute Command    sudo tedge config unset az.topics
```